Hide singleQueryPerAction (#850)
dataform-co/dataform: content/docs/guides/operations.md

These statements will be run without modification against the warehouse. You can use warehouse specific commands in these files, such as BigQuery's DML or DDL statements, or Redshift/Postgres specific commands.

<div className="bp3-callout bp3-icon-info-sign bp3-intent-none" markdown="1">
  BigQuery and SQL Data Warehouse run all operations for a file in the same context; the executed SQL is created by joining all operations with a semi-colon <code>;</code>.

>   For Redshift or Snowflake, operations are run as separate queries.

Reviewer: ```suggestion
  For other warehouse types, operations are run as separate queries.
```
